data/json/construction.json

```diff
@@ -59,6 +59,18 @@
     "pre_terrain": "t_pit",
     "post_terrain": "t_pit_glass"
   },
+  {
+    "type": "construction",
+    "id": "constr_pit_underground",
+    "group": "underground_pit",
+    "category": "DIG",
+    "qualities": [ [ { "id": "DIG", "level": 2 } ] ],
+    "required_skills": [ [ "survival", 3 ] ],
+    "time": "50 m",
```

Comment:
This is quite rapid.

Reply:
Upped to 300 minutes.  Is that more appropriate?

Reply:
It's 180 for a normal pit, I think this being long is appropriate.

Reply:
Should it be longer?

Reply:
I think it's fine as is, others probably have more informed opinions on the matter than I.